package.json

```diff
@@ -15,7 +15,7 @@
     "docs": "npx typedoc",
     "publish:release-candidate": "npx lerna publish --dist-tag next --preid rc --no-verify-access --conventional-prerelease bump preminor",
     "publish:release": "npx lerna publish --dist-tag latest --create-release github --no-verify-access --conventional-graduate bump minor",
-    "publish:hotfix": "npx lerna publish --no-verify-access --conventional-graduate bump patch",
+    "publish:hotfix": "npx lerna publish --dist-tag latest --no-verify-access --conventional-graduate bump patch",
```

Comment:
Don't think we should publish to GitHub like the release workflow does since that can result in tag conflicts. The `dist-tag` is the only diff I see that could explain why it wasn't working before. Will test and check.

Reply:
Actually, the problem appears to have been not fetching more git history. The `--dist-tag latest` bit is useful regardless since we'd want this hotfix publish to replace the previous release.